pkg/api/dtos/plugins.go

```diff
@@ -28,6 +28,7 @@ type PluginSetting struct {
 	SignatureType   plugins.SignatureType   `json:"signatureType"`
 	SignatureOrg    string                  `json:"signatureOrg"`
 	AngularDetected bool                    `json:"angularDetected"`
+	APIVersion      string                  `json:"apiVersion"`
```

Comment:
```suggestion
    // For backend plugins, this will be configured apiVersion or will default to v0alpha1
	APIVersion      string                  `json:"apiVersion,omitempty"`
```

Reply:
Same about the comment. Also, for the moment, the default is to leave it empty? At least for the DTO

Reply:
Looking more -- for this to be useful we will need the group/version.  We could either set that as the string, or in a more explicit struct like:
https://github.com/grafana/grafana/pull/87639/files#diff-0f03a03daf993e53c60c29d5e1d17007ae6f8a742065706640933077b6822745R56